### PR TITLE
Added command to "Refresh file contents"

### PR DESCRIPTION
**Pull request by ollitanska**
This PR will add a new command "Refresh file contents". This command will be shown alongside with the "Import and Compile" entry in the VSCode explorer right click menu. This command will "refresh" the selected file(s) contents by exporting it from the currently connected server to the local file system. This is useful when one modifies the production and want to synchronise the changes from the server to local file system with ease.

### CODE REVIEW

**Comment by ollitanska** [comment]
> @ollitanska Thanks for the PR! Can you explain a little more why this PR is necessary? A user can re-export from the ObjectScript Explorer or using the export command and we are trying to avoid adding too many menu options.
> 
> cc @gjsjohnmurray

I understand that this feature will overlap with the current functionality, but I still believe that this PR would be beneficial to the users. Tho, totally understand if you decide not to accept this PR.

When working with local files, a another user can simultaneously modify the source code at the server and your local files can easily get out of sync. This feature would add a really handy and a quick way to re-sync the local file system state with the server. For my dev setup, out of sync will happen regularly, if I or another person edits the production settings in the web browser and forgets to export the changes to ones' local file system.

The ObjectScript Explorer re-export way have some drawbacks for this kind of development style / setup:
- If the user has only some of the files in their local file system (and doesn't want them all), export will add extra files that they have to delete after export
- If the user has the files located in different kind of structure that what the export expects (e.g. files spread around in multiple submodules and thus folders), the exported files are not placed in the correct folders. (Yes, I know that you can create a custom mapping for this, but my proposed way would be way simpler)

Perhaps, this functionality could be improved to not only re-export the server source files to the local file system, but to compare the timestamps between the two, and automatically import & export based on which one is the newer. Or maybe this could be achieved by having a toggle to enable automatic file sync between local file system and the server.

**Comment by isc-bsaviano** [comment]
Thanks for taking the time to explain further. 

> When working with local files, another user can simultaneously modify the source code at the server and your local files can easily get out of sync.

Have you considered using a personal server for compilation and intellisense? This may not be possible in your case, but it's worth exploring since containerization and the community edition has made this much easier than in the past.

> The ObjectScript Explorer re-export way have some drawbacks for this kind of development style / setup:

These are two compelling reasons, with the caveat that web application files do need to be structured identically to the server for syncing them to work properly. I'm going to wait to see what John, one of our other maintainers, thinks before I dive into the code.

File: package.json
Comment (gjsjohnmurray) [inline package.json:479]:
```suggestion
          "when": "resourceScheme == file && editorLangId =~ /^objectscript/ && vscode-objectscript.connectActive",
```
Otherwise I think it would appear for isfs documents.

File: package.json
Comment (gjsjohnmurray) [inline package.json:142]:
```suggestion
          "when": "resourceScheme == file && editorLangId =~ /^objectscript/ && vscode-objectscript.connectActive"
```
I don't think the command should be offered for an isfs document.

File: package.json
Comment (gjsjohnmurray) [inline package.json:477]: My preference is for this command **not** to be added to the editor/context menu. That menu is already long, and users can't yet customize it. For people using a personal server namespace the command should be rarely needed. It's in Command Palette and a user can assign it a keyboard shortcut if they wish.

File: package.json
Comment (gjsjohnmurray) [inline package.json:565]: As commented re the editor/context menu, perhaps don't add this to explorer/context.

**Review by gjsjohnmurray — CHANGES_REQUESTED** [review]
I haven't yet looked at the actual implementation of the command.

File: src/extension.ts
Comment (isc-bsaviano) [inline src/extension.ts:794]: Since this command is only in the command palette now the implementation should be really simple, perhaps only a few lines:

```ts
const file = currentFile();
if (!file) {
  return;
}
loadChanges([file]);
```

IMO you also don't need to special Output channel messages anymore since you're only refreshing one file.

File: syntaxes/vscode-objectscript-output.tmLanguage.json
Comment (isc-bsaviano) [inline syntaxes/vscode-objectscript-output.tmLanguage.json:26]: You can remove these changes now that this syntax is no longer used. Otherwise, looks good!

**Comment by isc-bsaviano** [comment]
Thanks @ollitanska. Can you just run `npm run lint-fix` to resolve that code linting error? I will approve and merge after that.

**Review by isc-bsaviano — APPROVED** [review]

**Comment by isc-bsaviano** [comment]
Thanks again for the PR!

**Pull request merged**
